paasta_tools/cli/cmds/local_run.py

```diff
@@ -953,7 +956,8 @@ def configure_and_run_docker_container(
         return 1
 
     soa_dir = args.yelpsoa_config_root
-    volumes = list()
+    (mktemp_rc, mktemp_output) = _run("mktemp -d")
```

Comment:
would it be better to use https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory instead?